contents/customers/webshare.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How Webshare replaced Mixpanel, Hotjar, and FullStory with PostHog
+customer: Webshare
+logo: ../images/customers/webshare/webshare_logo.png
+logoDark: ../images/customers/webshare/webshare-logo-dark.png
+featuredImage: ../images/customers/webshare/featured.png
+industries:
+    - Devtool
+users:
+    - Marketing
+    - Leadership
+    - Customer Success
+toolsUsed:
+    - Experimentation
+    - Product analytics
+date: 2023-10-01
+---
+
+[Webshar](https://www.webshare.io/homepage) is a leading proxy server provider, enabling more than 100,000 users to connect using high-speed proxies in over 50 countries. Originally founded in 2018 by Utku Zihnioglu, Webshare has grown explosively and now serves over 250bn proxy requests each month — but retains the experimental, data-led approach it had in the early days. 
+
+“We have all our funnels nailed down, but we’re constantly running experiments on top of them,” says Utku, Webshare’s founder and CEO. “We’re really passionate about testing new ideas to get customers the best product we can.”
+
+In the early days the team used a mix of tools to facilitate this customer-centric approach — Mixpanel for analytics, Hotjar for session replays, plus a handful of internal tools to fill the gaps left by these platforms. 
+
+“We had to build our own tools for feature flags, because Mixpanel doesn’t do that,” says Utku. “We were using FullStory too…but we started looking for a replacement in 2021 because those tools just weren’t working for us anymore.”
```

Comment:
```suggestion
“We had to build our own tools for feature flags, because Mixpanel doesn’t do that,” says Utku. “We were using FullStory too... but we started looking for a replacement in 2021 because those tools just weren’t working for us anymore.”
```

Small request. Can you stop Google Docs from creating ellipses. Doesn't format properly in plain text.